### PR TITLE
feat(Organizing  the services folder)

### PR DESCRIPTION
**Pull request by anu-aj**
### Functionality:
Add folders in the services directory to separate each service

### Solution:
Folders created and organized

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
1. ng serve
2. firebase emulators:start

### CODE REVIEW

File: src/app/services/auth.service.ts [hunk 1]
```diff
@@ -5,9 +5,13 @@ import { User, UserAppSetting } from "../Interface/UserInterface";
...
 import { Observable } from 'rxjs';
+<<<<<<< HEAD
+import { BackendService } from './backEnd/backend.service';
+=======
 import { BackendService } from './backend.service';
 import { ThemeService } from './theme.service';
 import { map } from 'rxjs/operators';
+>>>>>>> 8e527f38f34408fb0a7dac1ace93c6502e5a26b8
 @Injectable({
   providedIn: 'root',
```
Comment (shivamgoyal7) [inline src/app/services/auth.service.ts:16]: solve the conflict

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]
solve the conflict

**Review by viv9k — CHANGES_REQUESTED** [review]
Please fix the spelling of backend folder in services.

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**